frontend/packages/db-structure/src/deparser/postgresql/operationDeparser.test.ts

```diff
@@ -344,4 +344,128 @@ describe('postgresqlOperationDeparser', () => {
       `)
     })
   })
+
+  describe('constraint operations', () => {
+    it('should generate ADD CONSTRAINT PRIMARY KEY statement', () => {
+      const operation: Operation = {
+        op: 'add',
+        path: '/tables/users/constraints/pk_users_id',
+        value: {
+          type: 'PRIMARY KEY',
+          name: 'pk_users_id',
+          columnName: 'id',
+        },
+      }
+
+      const result = postgresqlOperationDeparser(operation)
+
+      expect(result.errors).toHaveLength(0)
+      expect(result.value).toMatchInlineSnapshot(`
+        "ALTER TABLE \"users\" ADD CONSTRAINT \"pk_users_id\" PRIMARY KEY (\"id\");"
+      `)
+    })
+
+    it('should generate ADD CONSTRAINT FOREIGN KEY statement', () => {
+      const operation: Operation = {
+        op: 'add',
+        path: '/tables/orders/constraints/fk_orders_user_id',
+        value: {
+          type: 'FOREIGN KEY',
+          name: 'fk_orders_user_id',
+          columnName: 'user_id',
+          targetTableName: 'users',
+          targetColumnName: 'id',
+          updateConstraint: 'CASCADE',
+          deleteConstraint: 'SET_NULL',
+        },
+      }
+
+      const result = postgresqlOperationDeparser(operation)
+
+      expect(result.errors).toHaveLength(0)
+      expect(result.value).toMatchInlineSnapshot(`
+        "ALTER TABLE \"orders\" ADD CONSTRAINT \"fk_orders_user_id\" FOREIGN KEY (\"user_id\") REFERENCES \"users\" (\"id\") ON UPDATE CASCADE ON DELETE SET_NULL;"
```

Comment:
The correct SQL should be:

```suggestion
        "ALTER TABLE \"orders\" ADD CONSTRAINT \"fk_orders_user_id\" FOREIGN KEY (\"user_id\") REFERENCES \"users\" (\"id\") ON UPDATE CASCADE ON DELETE SET NULL;"
```

We should handle deleteConstraint: SET_NULL correctly by converting it to SET NULL in SQL.

Reply:
>  It would be good to write a test that parses with pg-query-emscripten.
> Can I merge this PR and fix it later, 

👍  looks good!